### ISSUE TITLE
Ruff python formatter builtin is outdated & duplicated

### ISSUE THREAD

**Issue by jakenvac** [nvimtools/none-ls.nvim#40]
### FAQ

- [X] I have checked the [FAQ](https://github.com/jose-elias-alvarez/null-ls.nvim#faq) and it didn't resolve my problem.

### Issues

- [X] I have checked [existing issues](https://github.com/jose-elias-alvarez/null-ls.nvim/issues?q=is%3Aissue) and there are no issues with the same problem.

### Neovim Version

v0.9.1

### Dev Version?

- [X] I am using a stable Neovim release version, or if I am using a dev version of Neovim I have confirmed that my issue is reproducible on a stable version.

### Operating System

macos, linux & windows

### Minimal Config

```Lua
n/a
```

### Steps to Reproduce

configure the ruff linter with `builtins.formatting.ruff` and try to format. Nothing happens.

### Reproducibility Check

- [ ] I confirm that my minimal config is based on the `minimal_init.lua` template and that my issue is reproducible by running `nvim --clean -u minimal_init.lua` and following the steps above.

### Expected Behavior

The document should be formatted.

### Actual Behavior

The wrong command is executed & the document does not format.

The command that is run is `ruff --fix` where it should be `ruff format`.

### Debug Log

n/a

### Help

Yes

### Requirements

- [X] I have read and followed the instructions above and understand that my issue will be closed if I did not provide the required information.

**Comment by mochaaP** [comment]
Reopening as per <https://github.com/nvimtools/none-ls.nvim/pull/41#issuecomment-1845466894>

**Comment by jakenvac** [comment]
@bellini666 After the confusion around this last week - are you able to confirm if there's an issue here at all? I only ask yourself as you're more familiar with Ruff.

I'd be happy to close it if it works fine for you.

**Comment by bellini666** [comment]
@jakenvac I don't think we have an issue here, everything is working fine for me!

But based on the fact that you were having issues, what were those? Maybe it was due to lack of documentation? Even the fact that you thought `ruff_format` shouldn't exist tells me that we probably should be documenting it better (i.e. telling why it exists alongside `ruff` itself)

**Comment by jakenvac** [comment]
That's a really good point. The issues I experienced I have chalked up to my lack of python knowledge (and it's ecosystem/tools) and the documentation for ruff not being very clear.

For context: I had looked through the `help` command from ruff and it's Github page(s) and couldn't find the distinction between the various 'formatting' commands. I probably could have spent more time looking.

So the actual issue I was experiencing was that when running either of the formatters we have configured for null-ls on some python code, nothing happened. I purposely used some poorly formatted python code to try this.
After I changed the command in null-ls, it did successfully 'format' the document.

All that to say, since you're happy that both tools are working, I'll close this and if anyone else experiences issues we can revisit.
